client/index.html

```diff
@@ -243,6 +243,12 @@ <h1 class="title">Settings</h1>
 									Show quits
 								</label>
 							</div>
+							<div class="col-sm-6">
+								<label class="opt">
+									<input type="checkbox" name="away">
+									Show when users go away  
```

Comment:
As opposed to @dgw's opinion, I think we should go with consistency, therefore something like `Show away/back changes` or the likes. It's not pretty, but it sticks out compared with the other options right now.